### PR TITLE
Security buff from cdb? gasp.

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
Marshal/shield wardrobe have balaclavas again

WO once again has the choice of the Osprey and SWAT as their mainarms, no longer has the option of the thing

Sgt now has basic medical access + sensors access, same as corpsmen

Restores the Triage to the list of possible loadout weapons for Corpspersons, now they can pick between three kinda meh SMGs :)

Also fixes a minor issue where the Longarm could be disassembled abut not put back together. Longarm, Osprey, Triage now may not be disassembled as they are far too complex to pull apart with wirecutters of all things.

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/automatic/battlerifle.dm [hunk 1]
```diff
@@ -73,6 +73,7 @@
 	item_state = "DMR"
 	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_PLASTIC = 15, MATERIAL_GLASS = 10)
 	mag_well = MAG_WELL_HRIFLE
+	gun_parts = null
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/automatic/battlerifle.dm:76]: dont put null, have it be like wood and plasteel spawns

File: code/modules/projectiles/guns/projectile/automatic/triage.dm [hunk 2]
```diff
@@ -18,6 +18,7 @@
 	price_tag = 900
 	penetration_multiplier = 1.2
 	init_recoil = SMG_RECOIL(0.6)
+	gun_parts = null //she's a delicate, complicated lil flower. Don't try to pull her apart.
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/automatic/triage.dm:21]: have it give scrap metal or something null is bad!

Comment (cdb-is-not-good) [reply code/modules/projectiles/guns/projectile/automatic/triage.dm:21]: guns with null parts will give a special message about being too complicated to disassemble

**Comment by cdb-is-not-good** [comment]
image
approved by SKinkk

**Pull request merged**